test/test_view_ops.py

```diff
@@ -918,28 +918,37 @@ def _test_ravel(tensors, size, nc=False):
                 self.assertEqual(flat.shape, torch.Size([size]))
                 self.assertEqual(src.view(-1), flat)
```

Comment:
Would it make sense to check against `np.ravel`?

cc: @mruberry

Reply:
Hi, @kshitij12345!
I tried to test this case locally in my system. I used the following test case: (same for non-contiguous inputs)
```python
self.assertEqual(flat, np.ravel(src.cpu()))
```
I got the following result:
```python
Traceback (most recent call last):
  File "/home/khushi/Documents/opensource/pytorch/test/test_view_ops.py", line 962, in test_ravel
    _test_ravel(tensors, 625)
  File "/home/khushi/Documents/opensource/pytorch/test/test_view_ops.py", line 918, in _test_ravel
    self.assertEqual(flat, np.ravel(src.cpu()))
  File "<__array_function__ internals>", line 180, in ravel
  File "/home/khushi/anaconda3/envs/torch-dev/lib/python3.10/site-packages/numpy/core/fromnumeric.py", line 1859, in ravel
    return asanyarray(a).ravel(order=order)
  File "/home/khushi/Documents/opensource/pytorch/torch/_tensor.py", line 721, in __array__
    return self.numpy()
TypeError: Got unsupported ScalarType QUInt8
```
I think this is because numpy does not support `quantized int`. [xref](https://github.com/pytorch/pytorch/blob/69c9bc9d161e15034e3ef2ff0f53193a408b72e0/torch/csrc/utils/tensor_numpy.cpp#L242)

What do you think, do we need to add this case for quantized dtypes? Thanks :)

Reply:
For quantized, I think we can use uous().view(-1)`. For supported dtypes use `np.ravel(src.cpu().numpy())`.

wdyt @mruberry?

Reply:
I am surprised this test is run with quantized types. I think it's OK not to use `np.ravel()` here but we should add it as a reference for the `ravel` OpInfo

Reply:
Sure, added :)